bundles/FileExplorerBundle/public/js/explorer.js

```diff
@@ -11,8 +11,8 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.settings.fileexplorer.explorer");
-pimcore.settings.fileexplorer.explorer = Class.create({
+pimcore.registerNS("pimcore.fileexplorer.explorer");
+pimcore.fileexplorer.explorer = Class.create({
```

Comment:
In the glossary bundle we try to keep the settings, maybe we can do this here too?